### PR TITLE
Upgrade: upgrade BTG to v0.21

### PR DESCRIPTION
**Pull request by www222fff**

### CODE REVIEW

**Comment by zhenangroup** [comment]
I am pretty excited to leave a comment for the first time and I will resolve this issue within 15 days.

**Comment by qasimsaleem1** [comment]
Waiting for full gitian release + phala bridge LIVE

**Comment by qasimsaleem1** [comment]
Waiting for BTG to v0.21 @www222fff When we can expect release @h4x3rotab

**Comment by www222fff** [comment]
> 

Coding review now, once finish will soon

**Comment by qasimsaleem1** [comment]
@www222fff @h4x3rotab Waiting for v0.21 release any estimation now?

**Comment by qasimsaleem1** [comment]
Waiting for 0.21 release in April? If not complete at least waiting for Beta in April @www222fff @h4x3rotab 
Waiting for your kind response @www222fff @h4x3rotab

**Comment by qasimsaleem1** [comment]
What happened waiting for 0.21 release? @www222fff @h4x3rotab

**Comment by Jamesfrancs** [comment]
We don't understand why it is taking much time. Kindly launch in MAY 2022 at least Beta release. Its already being started from DEC 2021 @h4x3rotab @www222fff .
Waiting For 0.21 and btg phala bridge
Waiting for your response.

**Comment by www222fff** [comment]
@Jamesfrancs @qasimsaleem1 As code review need support so delayed, could you support review and comments?

**Review by PcjsCorp — APPROVED** [review]

File: contrib/testgen/convert_btc_to_btg_bech32
Comment (MentalNomad) [inline contrib/testgen/convert_btc_to_btg_bech32:1]: Is this just a residual file remaining from running test code? Doesn't appear to belong here.

Comment (www222fff) [reply contrib/testgen/convert_btc_to_btg_bech32:1]: Yes, binary it is should be removed.
is that still request for upgrade v21 or not? Activity is launched already three years before but then paused later.

Comment (MentalNomad) [reply contrib/testgen/convert_btc_to_btg_bech32:1]: It's now un-paused; I've just completed code review.

Comment (www222fff) [reply contrib/testgen/convert_btc_to_btg_bech32:1]: really? Hope @h4x3rotab aware this then i can merge this PR.

**Comment by MentalNomad** [comment]
Updating to 21; ready for full testing and final tweaks.

**Pull request merged**